aspnetcore/signalr/client-features.md

```diff
@@ -14,11 +14,19 @@ uid: signalr/client-features
 
 The table below shows the features and support for the clients that offer real-time support.
 
-| Feature | .NET Core | JavaScript | Java |
+| Feature | .NET | JavaScript | Java |
```

Comment:
Also, the client is for .NET Standard and supports .NET Framework, Xamarin, .NET Core, Unity, etc. So calling it ".NET Core" isn't really appropriate. Changed to `.NET`